File: Euclid/Messages/Outgoing/HEIGHTMAP.cs
using Euclid.Network.Streams.Util;

namespace Euclid.Messages.Outgoing
{
    class HEIGHTMAP : IMessageComposer
    {
        private string heightmap;

        public HEIGHTMAP(string heightmap)
        {
            this.heightmap = heightmap;
        }

        public override void Write()
        {
            Data.Add(new ArgumentEntry(this.heightmap));
        }
    }
}

File: euclid/src/main/java/euclid/messages/outgoing/HEIGHTMAP.java
package euclid.messages.outgoing;

import euclid.messages.IMessageComposer;
import euclid.network.streams.util.ArgumentEntry;

public class HEIGHTMAP extends IMessageComposer {

    private final String heightmap;

    public HEIGHTMAP(String heightmap) {
        this.heightmap = heightmap;
    }

    @Override
    public void write() {
        getData().add(new ArgumentEntry(heightmap));
    }
}
